### DISCUSSION TITLE
Getting ReactNativeFirebaseMessagingHeadlessTask Error

### DISCUSSION THREAD

**Discussion by anilkumar2273702** [invertase/react-native-firebase#5376] [Q&A]
I'm getting the following error - WARN  No task registered for key ReactNativeFirebaseMessagingHeadlessTask
// index.js
import {AppRegistry} from 'react-native';
import App from './App'; 
import {name as appName} from './app.json';
import 'react-native-gesture-handler';

export async function firebaseBackgroundMessage(message) {
  console.log(message);
  return Promise.resolve();
}

AppRegistry.registerComponent(appName, () => App);
AppRegistry.registerHeadlessTask(
  'RNFirebaseBackgroundMessage',
  () => firebaseBackgroundMessage,
);

I have done the above code in index.js, but still getting errors. When I open a notification from the background in Android mobile.

**Comment by mikehardy** [comment]
It sounds like you have not integrated react-native-firebase correctly somehow. the ReactNativeFirebaseMessagingTask is added automatically for you I think during this module's startup sequence. In your code you are simply passing it a method to call?

https://github.com/mikehardy/rnfbdemo/blob/master/make-demo.sh - example of proper / basic integration

**Reply by anilkumar2273702** [reply]
ok, thanks for your response @mikehardy, It's resolved now.
"RNFirebaseBackgroundMessage" to "ReactNativeFirebaseMessagingHeadlessTask"

**Reply by chaulapatrice** [reply]
@anilkumar2273702 how did you resolve it?
